src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/Runners/MonitorRunner.cs

```diff
@@ -261,7 +267,11 @@ public async Task WriteUserSettingsAsync(RootOptions options, CancellationToken
 
             JsonSerializerOptions serializerOptions = new()
             {
```

Comment:
I couldn't find it but isn't there a particular setting in `JsonSerializerOptions` that is required security?

Reply:
Wait, nevermind, this is only a serializer, the rules don't apply.